app/frontend/src/components/Bar/BarWithHelp.tsx

```diff
@@ -26,7 +26,7 @@ export default function BarWithHelp({
   description,
 }: BarWithHelpProps) {
   const classes = useStyles();
-  return (process.env.REACT_APP_API_BASE_URL ? <></> :
+  return process.env.IS_BETA_ENABLED ? null : (
```

Comment:
Should it not be the other way around instead? This is currently saying hide the bars if beta is enabled 🤔

Reply:
Or is beta meant the "soft launch" in this case?

Reply:
Yep, I tested it earlier and they disappeared but I was using the wrong env var